src/Hl7.Fhir.Support.Poco/Model/Instant.cs

```diff
@@ -93,8 +94,8 @@ public static Instant Now()
             return new Instant(DateTimeOffset.Now);
         }
 
-        public Primitives.PartialDateTime? ToPartialDateTime() => 
-            Value != null ? (Primitives.PartialDateTime?)Primitives.PartialDateTime.FromDateTimeOffset(Value.Value) : null;
+        public P.DateTime ToPartialDateTime() => 
```

Comment:
Rename to ToDateTime()

Reply:
I considered it, but thought it was too big a change on the interface. Then again, the name of the returned type has changed, so that's probably a breaking change to hahaha.  Well, if we've both considered changing it, I'll do it then.

Reply:
Impossible - that method already exists and returns a System.DateTime.  It becomes really confusing for the user if you are not used to be working with the "system" types to get one in return.  This let me to conclude that we should not offer these functions at all. They should be approached from the other side: if you work with system types you know how to derive them from the FHIR types.